src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -285,12 +340,8 @@ private void updateCAType(Topic topic) {
                 String caType = caTypeList.get(0);
                 certificateManager.update(getPassphrase(), CertificateStore.CAType.valueOf(caType));
             }
-
-            List<String> caCerts = certificateManager.getCACertificates();
```

Comment:
why aren't we doing this during startup anymore?

Reply:
Updated.